doc/script_commands.txt

```diff
@@ -5622,6 +5625,37 @@ Examples:
 
 ---------------------------------------
 
+*equipidx(<index>{,<item id>})
+
+The equipidx function will attempt to equip the item at the given inventory
+inventory index. This feature is only really useful when multiple instances
+of an Item ID exist in the player's inventory.
+
+If item id is specified, the item id at the specified index must be of that
+item id.
+
+Returns 1 if the item was successfully equipped.
+Returns 0 if the item failed to equip.
+
+See getinventorylist() for an explanation on retrieving an inventory index.
+
+Examples:
+
+//This will attempt to equip the item at inventory index 45.
+	equipidx(45);
+
+//This will attempt to equip a 1201 (Knife) every 1 second until it
+//has checked every inventory item.
+	getinventorylist();
+	for (.@i = 0; .@i < ; .@i++) {
+		if (@inventorylist_id[.@i] == Knife) {
+			equipidx(.@i, Knife);
+			sleep2(1000);
```

Comment:
this is consider a trick, and shouldn't include in the documentation
inside the documentation has to teach proper scripting methods,
so no sleep2 inside a refiner script
example: what happens if the player disconnect during the refining process ?

Just remove the examples will do

`for (.@i = 0; .@i < ; .@i++) {` <--- missing `@inventorylist_count` ... nonono just remove this example